doc/source/ray-core/scheduling/labels.md

```diff
@@ -0,0 +1,130 @@
+---
+description: "Learn about using labels to control how Ray schedules tasks, actors, and placement groups to nodes in your Kubernetes cluster."
+---
+
+# Use labels to control scheduling
+
+In Ray version 2.48.0 and above, you can use labels to control scheduling for KubeRay. Labels are supported as an alpha feature.
+
+This page provides a conceptual overview and usage instructions for labels. Labels are key-value pairs that provide a human-readable configuration for users to control how Ray schedules tasks, actors, and placement group bundles to specific nodes.
+
+Ray labels build upon Kubernetes labels, including formatting restrictions. See the [Kubernetes docs on labels and selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
+
+## How do labels work?
+
+The following is a high-level overview of how you use labels to control scheduling:
+
+- Ray sets default labels that describe the underlying compute. See [](defaults).
+- You define custom labels as key-value pairs. See [](custom).
+- You specify *label selectors* in your Ray code to define label requirements. You can specify these requirements at the task, actor, or placement group bundle level. See [](label-selectors).
+- Ray schedules tasks, actors or placement group bundles based on the specified label selectors.
+- If you're using a dynamic cluster with autoscaler V2 enabled, the cluster scales up to add new nodes from a designated worker group to fulfill label requirements.
+
+(defaults)=
+## Default node labels 
+
+During cluster initilization or as autoscaling events add nodes to your cluster, Ray assigns the following default labels to each node:
+
+| Label | Description |
+| --- | --- |
+| `ray.io/node-id` | A unique ID generated for the node. |
+| `ray.io/accelerator-type` | The accelerator type of the node, for example `L4`. CPU-only machines have an empty string. |
+| `ray.io/market-type` | Indicates whether the node uses `spot` instances or `on-demand` instances. |
+| `ray.io/node-group` | The name of the node worker group or `head` for the head node. You specify group names while configuring worker groups. See [](kuberay-config). |
+| `ray.io/region` | The cloud region of the node. | 
+| `ray.io/availability-zone` | The available zone of the node. |
+
+.. note:: You can override default values using `ray start` parameters.
+
+The following are examples of default labels:
+
+```python
+"ray.io/market-type": "spot" # Default label specifying the node is a spot instance.
+"ray.io/accelerator-type": "" # Default label indicating the machine is CPU-only.
+```
+
+(custom)=
+## Define custom labels
+
+You can add custom labels to your nodes using the `--labels` or `--labels-file` parameter when running `ray start`. See the following examples:
+
+<!-- INSERT EXAMPLES -->
```

Comment:
@janetli19 @MengjinYan Still haven't really seen much here beyond what you can accomplish with defaults.

Maybe an example where we mark something like CPU size to allow users to specify tasks to specific VM types in a heterogenous CPU environment?

Could use help coding something up as I'm pretty novice at Ray.

Reply:
Just noticed that's exactly the example we give below. So here we just need to show define the config for that value?